### ISSUE TITLE
Csv file is not generated properly

### ISSUE THREAD

**Issue by APSathishKumar** [alexMyG/AndroPyTool#14]
Csv generated as a use of argument -fw is filled with only zeros in both rows and columns,whereas when argument -csv EXPORTCSV is used csv file is not generated stating that flowdroid error(screen shot added). please help me
![IMG_20191001_185002](https://user-images.githubusercontent.com/56039044/65982514-23026700-e499-11e9-88f1-59285e6f0f93.jpg)

**Comment by iamrjkunal** [comment]
Hi @APSathishKumar ,
I am also getting the same error. Do you know the solution now?

**Comment by harrypro02** [comment]
did u get the solution?

**Comment by APSathishKumar** [comment]
I am not sure about the updated version but it didn't generate CSV file for me. So what I did is referred to theirs and also some other research papers, found out some features and developed a python program which extracted the necessary features from droidbox output(which will be in JSON format) and store those features as a CSV file.

**Comment by harrypro02** [comment]
yes, i think they haven't fixed the errors. can u share me the thing which u generated i will jst have a try

**Comment by Jesbinder** [comment]
I am still facing the same issue, tried to convert **JSON to CSV** format but still, the CSV format data is scrambled. If anybody got the solution, do let me know. 
Thanks!
